### PR TITLE
fix: added fallback for image error

### PR DESCRIPTION
**Pull request by boidushya**
# Description

Fallback image on image error. Currently implemented for notification items, can be adopted for all external image assets in future

# Type of change

Please put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Fixes/Resolves (Optional)

#69  (partially)

# Examples/Screenshots (Optional)

image

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/notifications/AppExplorer/AppCard/AppCard.scss [hunk 1]
```diff
@@ -44,6 +44,7 @@
       color: var(--fg-color-2);
     }
     &__subscribe {
+      margin-top: 0.67em;
```
Comment (devceline) [inline src/components/notifications/AppExplorer/AppCard/AppCard.scss:47]: Why do we need this

File: src/components/notifications/AppExplorer/AppCard/index.tsx [hunk 2]
```diff
@@ -55,6 +56,8 @@ const AppCard: React.FC<AppCardProps> = ({ name, description, logo, bgColor, url
     [userPubkey, name, description, logo, bgColor, url]
   )
 
+  const imgRef = React.useRef(null)
```
Comment (devceline) [inline src/components/notifications/AppExplorer/AppCard/index.tsx:59]: What are we using the ref for

Comment (boidushya) [reply src/components/notifications/AppExplorer/AppCard/index.tsx:59]: Oops, my bad, was trying something else with this, will remove this

File: src/components/notifications/AppExplorer/AppCard/AppCard.scss (hunk 1 above)
Comment (boidushya) [reply src/components/notifications/AppExplorer/AppCard/AppCard.scss:47]: image
its cuz of the spacing between the button and text

**Review by devceline — APPROVED** [review]
LGTM

**Pull request merged**